Kernel/Bus/USB/USBManagement.cpp

```diff
@@ -23,7 +23,7 @@ UNMAP_AFTER_INIT USBManagement::USBManagement()
 void USBManagement::register_driver(NonnullLockRefPtr<Driver> driver)
 {
     dbgln_if(USB_DEBUG, "Registering driver {}", driver->name());
-    s_available_drivers->append(driver);
+    s_available_drivers->try_append(driver).release_value_but_fixme_should_propagate_errors();
```

Comment:
did not check all things, but driver registration is quite early that I do not think error propagation
is necessary, as all of them should fit into the initial heap (I hope)

Reply:
For these driver registration functions I still want to return an error and use `MUST` on the call site (the static driver init function) in case we ever want to support loadable modules. `DeviceTree::Management::register_driver()` already works like this.